cmake/Install.cmake

```diff
@@ -9,7 +9,7 @@
 #elseif(APPLE)
 #set(CPACK_GENERATOR "IFW;TGZ")
 #elseif(UNIX)
-#set(CPACK_GENERATOR "STGZ;TGZ")
+#set(CPACK_GENERATOR "IFW;STGZ;TGZ")
```

Comment:
This is fine, also if these comments aren't needed, we could remove them entirely.